openmls/src/extensions/mod.rs

```diff
@@ -85,30 +88,34 @@ pub enum ExtensionType {
     /// Reserved. This must not be used.
     Reserved = 0,
 
+    /// The application id extension allows applications to add an explicit,
```

Comment:
Sorry, but these changes make it impossible to review.

Reply:
Yeah... that's why I proposed to make the cleanup first. Sorry for that.